tools/digester/main.go

```diff
@@ -25,6 +25,8 @@ import (
 	"encoding/csv"
 	"flag"
 	"fmt"
+	"github.com/containers/image/v5/types"
+	"github.com/opencontainers/go-digest"
```

Comment:
please move down - to not be with the standard library packages.

Reply:
done, thanks